msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -82,6 +72,16 @@ import com.microsoft.identity.common.java.providers.microsoft.azureactivedirecto
 import com.microsoft.identity.common.java.util.ResultFuture
 import com.microsoft.identity.common.java.util.StringUtil
 import com.microsoft.identity.common.java.nativeauth.util.checkAndWrapCommandResultType
+import com.microsoft.identity.nativeauth.statemachine.errors.ErrorTypes
```

Comment:
Shouldn't this PR also contain updated tests?

Reply:
This PR should contain updated tests. Previously the test branch had not been merged and the test files did not exist. Now it has been updated.